app/src/pages/Home/LanguagePicker.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Box,
+  Chip,
+  IconButton,
+  Link,
+  makeStyles,
+  Typography,
+} from "@material-ui/core"
+import AddCircleOutlineIcon from "@material-ui/icons/AddCircleOutline"
+import { SupportedLanguages } from "locales"
+import { languageNameFromCode } from "locales/functions"
+import * as React from "react"
+import { Trans, useTranslation } from "react-i18next"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    alignItems: "center",
+    display: "flex",
+    justifyContent: "center",
+    listStyleType: "none",
+    margin: `0 0 ${theme.spacing(1)}px`,
+    padding: 0,
+    "& > li": {
+      marginLeft: theme.spacing(0.5),
+      marginRight: theme.spacing(0.5),
+    },
+  },
+}))
+
+export const LanguagePicker = () => {
+  const classes = useStyles()
+  const { t, i18n } = useTranslation()
+
+  return (
+    <Box textAlign="center">
+      <ul className={classes.root}>
+        {SupportedLanguages.map((code) => (
+          <li key={code}>
+            <Chip
+              label={languageNameFromCode(code)}
+              variant="outlined"
+              color={code === i18n.language ? "primary" : "default"}
+              onClick={() => {
+                i18n.changeLanguage(code)
+              }}
+            />
+          </li>
+        ))}
+        <li>
+          <IconButton
+            component={Link}
+            size="small"
+            href="https://github.com/avimoondra/fishbowl#Localization"
+            target="_blank"
+          >
+            <AddCircleOutlineIcon />
+          </IconButton>
+        </li>
+      </ul>
+      <Typography color="textSecondary" variant="caption">
+        <Trans t={t} i18nKey="languagesPoweredBy">
+          {"Powered by our friends at "}
+          <Link href="https://locize.com" target="_blank">
+            {{ serviceName: "locize" }}
+          </Link>
+          .
+        </Trans>
+      </Typography>
+    </Box>
```

Comment:
@avimoondra, I just threw this ☝🏻 together to get something going:

<img width="577" alt="Screen Shot 2021-02-20 at 12 28 55 PM" src="https://user-images.githubusercontent.com/847532/108603696-77043900-7377-11eb-90c0-96c61f07dc41.png">

but certainly open to other options!

Reply:
This looks great.. 

For future reference, we can add:

mandarin chinese
hindi
spanish
french
Standard Arabic – 274 million speakers
Bengali – 265 million speakers
Russian – 258 million speakers
Portuguese – 234 million speakers
Indonesian – 199 million speakers
Urdu – 170 million speakers
Standard German – 132 million speakers
Japanese – 128 million speakers
Swahili – 98 million speakers
Marathi – 95 million speakers
Telugu – 93 million speakers
Western Punjabi – 93 million speakers
Wu Chinese – 82 million speakers
Tamil – 81 million speakers
Turkish – 80 million speakers

We should throw Italian in for good measure! 😄 


And some of our own data

![image](https://user-images.githubusercontent.com/2534903/108613710-5f05d700-73c2-11eb-94da-ea738e09d128.png)

Reply:
If we add a bunch of languages, the pills may not "scale" -- maybe we switch to drop down later. And eventually we can throw it in the "settings" gear icon / page toggled from bottom right corner during game play.